dashboard/lib/policies/child_account.rb

```diff
@@ -1,21 +1,25 @@
+require 'cdo/shared_constants'
+
 class Policies::ChildAccount
   # Values for the `child_account_compliance_state` attribute
   module ComplianceState
     # The student's account has been used to issue a request to a parent.
-    LOCKED_OUT = 'l'.freeze
+    LOCKED_OUT = SharedConstants::CHILD_ACCOUNT_COMPLIANCE_STATES.LOCKED_OUT
```

Comment:
This change feels odd, but the consensus was to be consistent with the way we pass constants to the front-end, which is via `shared_constants.rb`. We also don't want to reference anything in `/dashboard` from `/lib`, so there wasn't a great option for defining these constants in the policy if we want to pass them to the React app. I could also simply be missing an obvious better solution here, so please call it out if you have a better solution!